client/src/components/Project/Project.js

```diff
@@ -45,10 +45,17 @@ class Project extends Component {
     }
   }
 
+  getAppliedStatus () {
+    return (
+      this.props.user.opportunitiesAppliedFor &&
```

Comment:
Could we have `user.opportunitiesAppliedFor` defaulted to `[]` before it gets here? This may simplify things downstream such as this conditional.

Reply:
i figure that this check is kind of similar to checking if the usertype is 'volunteer'.. since only volunteers are supposed to have opportunitiesAppliedFor, though truth be told org contacts do also have this field as it is defaulted to [] when getting the user from the server with toJSON, even though mary doesn't have the field set at all in the db.. (this could be changed)

so technically the check could be removed altogether and it would work, but since a Project is rendered by ProjectList by Profile, and a Profile can belong to a contact who isn't supposed to have any opportunitiesAppliedFor in the first place, maybe this needs to be changed altogether? on top of this, projects probably shouldn't be rendering an 'x' or a 'check' at all for an org contact?

just my thoughts..